content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -0,0 +1,236 @@
+
+2. Click **New Project** and then select **Import a Prisma repository**.
+3. Fill in the repository and project details, and click **Next**.
+
+   <!-- TODO redo -->
+   ![Import a Project](./450-03-import-project.png)
+
+4. Connect the Prisma Data Platform to your database and set up the Data Proxy:
+   1. Paste your PostgreSQL connection string.
+   1. Under **Location**, select a Data Proxy location that is geographically close to your database location.
+   1. Click **Create Project** to test the connection and set up the Data Proxy.
+      
+   You are greeted with a new connection string that starts with `prisma://`. 
+5. Copy the `prisma://` connection string to your clipboard.
+
+   <!-- TODO redo -->
+   ![Data Proxy Page](./450-05-data-proxy.png)
+
+
+## 7. Set the Data Proxy Connection string in your environment
+
+With the Data Proxy connection string known, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
+
+As the Data Proxy Client does not read `.env` files by default, you also need to install `dotenv-cli` locally:
+
+```
+npm install dotenv-cli
+```
+
+You are now ready to test your script locally.
+
+## 8. Test your script locally
+
+Execute your script locally:
+
+```terminal
+npx dotenv -- deno run -A ./index.ts
+```
+
+Now you can open a browser to [http://localhost:8000/](http://localhost:8000/) and it will write your request to the database.
+
+```
+{
+  "id": 3,
+  "level": "Info",
+  "message": "GET http://localhost:8000/",
+  "meta": {
+    "headers": "{}"
+  }
+}
+```
+
+Refresh the page a couple times to verify that it's working. Now if you click on the **Data Browser** tab on your Prisma Cloud project, you should see `Info` logs written to your database.
+
+![Prisma Data Browser](./450-06-data-browser.png)
+
+It's working locally!
+
+## 9. Deploy to Deno Deploy
+
+Finally we are ready to also deploy our tiny Deno application to Deno Deploy:
+
+- [https://dash.deno.com/new](https://dash.deno.com/new)
+- Select your GitHub organization or user, repository and branch
+- Select the "Automatic" deployment mode
+- Select the `index.ts` as entry point
+- Make sure that the project has a good name
+- "Add Env Variable" to add `DATABASE_URL` with the Data Proxy connection string
+- Click "Link" button
+- Watch the deployment happen
+- When it is done, it will redirect to a project view
+- Click the blue "View" button on the top right to open your Deno Deploy deployment
+- See browser window showing same as locally
+
+Visit your deployment URL and you'll again see:
+
+```
+{
+  "id": 5,
+  "level": "Info",
+  "message": "GET https://prisma-deno-deploy.deno.dev/",
+  "meta": {
+    "headers": "{}"
+  }
+}
+```
+
+You're all set! You've successfully deployed a Deno application written in TypeScript that uses Prisma to talk to your PostgreSQL database.
+
+Give yourself a pat on the back, you deserve it!
```

Comment:
I don't really like this line for a tech doc, but until we update the other similar guides (e.g. Cloudflare), I think it's OK to leave it.

Reply:
Same here.